### PR TITLE
Added pi and e constants

### PR DESCRIPTION
**Pull request by SpeckyYT**

### CODE REVIEW

**Comment by zTags** [comment]
very cool

File: libraries/std/lib.spwn [hunk 1]
```diff
@@ -34,6 +34,9 @@ return {
     ..events,
     ..ctrl_flow,
 
+    pi: constants.pi,
+    e: constants.e,
```
Comment (camila314) [inline libraries/std/lib.spwn:38]: one-letter variable in global space, no bueno.

**Review by camila314 — CHANGES_REQUESTED** [review]
e

**Comment by SpeckyYT** [comment]
`e` got renamed to `euler` now

**Comment by camila314** [comment]
good enough

**Pull request merged**